release/serve_tests/workloads/serve_resnet_benchmark.py

```diff
@@ -0,0 +1,167 @@
+import ray
+from ray import serve
+from torchvision import models
+from typing import List
+import torch
+from ray.serve.drivers import DAGDriver
+from ray.dag.input_node import InputNode
+import asyncio
+import aiohttp
+import starlette
+import time
+from serve_test_utils import save_test_results
+import numpy as np
+import click
+from typing import Optional
+from ray.serve.handle import RayServeHandle
+
+
+# 8 images as input when batch size increase, we replica the input here
+input_uris = [
+    "http://images.cocodataset.org/test-stuff2017/000000000019.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000128.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000171.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000184.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000300.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000311.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000333.jpg",
+    "http://images.cocodataset.org/test-stuff2017/000000000416.jpg",
+]
+
+
+@serve.deployment
+class ImageObjectioner:
+    def __init__(self, handle: RayServeHandle, device="cpu"):
+        self.model = models.resnet50(pretrained=True)
+        self.model.eval().to(device)
+        self.device = device
+        self.handle = handle
+
+    async def predict(self, uris: List[str]):
+
+        preprocessing_tasks = []
+        for uri in uris:
+            preprocessing_tasks.append(await self.handle.remote([uri]))
+        image_tensors_lists = await asyncio.gather(*preprocessing_tasks)
+        image_tensors = [
+            tensor for item_tensors in image_tensors_lists for tensor in item_tensors
+        ]
+        data = torch.cat(image_tensors).to(self.device)
+        start = time.time()
+        res = self.model(data).to("cpu")
+        end = time.time()
+        return {"result": res, "model_inference_latency": end - start}
+
+
+@serve.deployment(
+    ray_actor_options={"runtime_env": {"pip": ["validators"]}}, num_replicas=5
```

Comment:
Out of curiosity any reason not to include this in the `cluster_env` for this test instead of `runtime_env`?